### PR TITLE
Awesome books: With  ES6

### PR DESCRIPTION
**Pull request by Sonickmumba**
In this pull request, I

- Have restructured the Awesome book app code to make it more organized by using modules.

### CODE REVIEW

File: modules/bookActions.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
```
Comment (zhadier) [inline modules/bookActions.js]: - [ ] Kindly remove this `eslint-disable` command in all of your folders. Eslint checks exist to ensure that you follow best JavaScript practices while coding. It is the industry standard and any company has their own coding norms which you must follow so kindly remove this disable check and fix your files accordingly

File: modules/books.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+/* eslint-disable */
```
Comment (zhadier) [inline modules/books.js]: - [ ]  Kindly remove `eslint-disable` from here aswell.

File: modules/menuEvents.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+//     e.preventDefault()
+//     menuNavMobile.classList.remove('show');
+//     body.classList.remove('show');
+//     page.classList.remove('show');
+//     menuBar.classList.remove('show');
+//     bookWrapper.style.display ='none';
+//     formWrapper.style.display = 'none';
+//     contactWrapper.style.display = 'flex';
+//   });
+// }
```
Comment (zhadier) [inline modules/menuEvents.js]: - [ ]  Please refrain adding commented out code in your workspace as it is frowned upon and reduces maintainability and manageability. Kindly remove this code here.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @Sonickmumba  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+/modules/books.js
+/modules/bookActions.js
```
Comment (zhadier) [inline .eslintignore]: - [ ] As I've requested before, kindly ensure that `eslint checks` are always running in the modules that you create. There's always a way to make the Linters work and they'll seem hard the first time, but once you get in the habit of following the practices you'll see that they're indeed helpful and will make you write much better code. [Here is a resource](https://sourcelevel.io/blog/what-is-a-linter-and-why-your-team-should-use-it) that shows the many advantages of using linters. Please remove these modules in the `.eslintignore` file and fix the linter errors in your modules accordingly.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @Sonickmumba  👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🌤️ 



 ## Highlights

✔️ No linter errors
✔️ GitFlow was setup correctly
✔️ Professional pull request title and description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Sonickmumba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**